user/languages/php.md

```diff
@@ -62,8 +62,21 @@ php:
 versions used in your build, consult "Build system information" in the build
 log.
 
-> Specifying exact versions like 5.3.8 is discouraged as it may break your build
-when we update PHP versions on Travis CI.
+These may not be the most recent releases. If you need to ensure the use of most recent releases, do provide the third number; e.g.,
+
+```yaml
+language: php
+php:
+  - 7.1.9
+```
+{: data-file=".travis.yml"}
+
+### PHP 5.2(.x) and 5.3(.x) support
```

Comment:
What do you think about removing this section from this file entirely, and just leaving the info in Precise / Trusty?

Reply:
I think it is important there is something being said here, very much like HHVM is treated. I expect users of PHP to look for information on this page, rather than to look at Precise/Trusty .

Reply:
And I'd probably prefer "we don't support" over "we're unable to provide support". Maybe @MariadeAnton has thoughts on this?

Reply:
Aye. That indeed sounds better.